iroh/src/lib.rs

```diff
@@ -1,5 +1,79 @@
 //! Send data over the internet.
 //!
+//! # Getting started
+//!
+//! ## Example
+//!
+//! Create a new node and add some data to the blobs store. This data will be
+//! available over the network.
+//!
+//! ```rust
+//! # async fn run() -> anyhow::Result<()> {
```

Comment:
wait, what?  We can just write `async fn` in doc tests and it'll work?  How did I now know that?